packages/core/src/components/css-grid/css-grid.tsx

```diff
@@ -103,8 +109,8 @@ export class CssGrid {
   }
 
   render() {
-    const style = {};
-    if (this.currentTemplate?.length !== 0) {
+    const style: Record<string, string> = {};
+    if (this.currentTemplate && this.currentTemplate.length !== 0) {
```

Comment:
Is there a specific reason why this was changed? I prefer using the optional chaining operator.